syft/frameworks/torch/tensors/interpreters/replicated_shared.py

```diff
@@ -146,6 +244,37 @@ def get_shares_map(self):
         """
         return self.child
 
+    def apply_to_shares(self, function, *args, **kwargs):
+        """
+        function: a reference to a function
+        e.g. torch.Tensor.view, torch.nn.F.fold
+        """
+        shares_map = self.get_shares_map()
+        players = self.get_players()
+        shares_map = {
+            player: (
+                function(shares_map[player][0], *args, **kwargs),
+                function(shares_map[player][1], *args, **kwargs),
+            )
+            for player in players
+        }
+        return ReplicatedSharingTensor(shares_map)
+
+    def apply_to_shares_(self, function, *args, **kwargs):
```

Comment:
Q: Is this used?

Reply:
No, I thought I will need it for convolution, but I didn't have to do any in-place operations, should I remove it? 
I think I should remove it, but I'm like 99% sure I'll need it in the future and I'll regret removing it 😂

Reply:
removed it